flow360/component/flow360_params/flow360_params.py

```diff
@@ -132,6 +133,36 @@ class SubsonicInflow(Boundary):
     rampSteps: PositiveInt
 
 
+class SupersonicInflow(Boundary):
+    """:class:`SupersonicInflow` class for specifying the full fluid state at supersonic inflow boundaries
+
+    Parameters
+    ----------
+    Density : PositiveFloat
+        Fluid density (non-dimensional). Default freestream fluid density in Flow360 is 1.0.
+
+    Velocity: Tuple[float, float, float]
+        Fluid velocity (non-dimensional), normalized to Mach number.
+    Pressure: PositiveFloat
+        Fluid pressure (non-dimensional). Default freestream fluid pressure in Flow360 = 1.0/gamma = 1.0/1.4.
+                                                                circumferentialForcePerArea[SI=N/m2]/rho_inf/C_inf^2
```

Comment:
this line should be removed (circumferentialForcePerArea)

Reply:
removed